client/state/checklist/reducer.js

```diff
@@ -32,6 +41,24 @@ export const items = createReducer(
 	itemSchemas
 );
 
+export const showChecklistNotification = createReducer(
+	false,
+	{
+		[ SITE_CHECKLIST_NOTIFICATION ]: ( state, action ) => action.bool,
+	},
+	checklistNotificationSchema
+);
+
+export const nextChecklistTask = createReducer(
+	'',
+	{
+		[ SITE_CHECKLIST_NEXT_TASK ]: ( state, action ) => action.taskId,
+	},
+	nextChecklistTaskSchema
+);
+
 export default combineReducers( {
 	items,
+	showChecklistNotification,
+	nextChecklistTask,
```

Comment:
I'm not sure we need these new reducers and actions. If I'm not wrong, we should be able to make the new feature work without adding new redux entries. Could you tell me why we need those ones?

Reply:
Thanks for the review Taegon! This is a great question. The honest answer is I'm not sure if we do; I wasn't sure how to make the state persist across multiple Components and pass it between them (`InlineHelp` and `InlineHelpPopover`), and in all my experimentation this is what worked. I'm happy to be shown a more efficient way!

Reply:
I see what you've been struggling with. Sharing states across multiple components tend to be tricky but it, fortunately, would be much easier where the components have a parent-child relationship.
For this case, as I wrote below, a simpler way is passing props to the child component, `InlineHelpPopover` from the parent component `InlineHelp`.

Reply:
Thank you! I had a lightbulb moment yesterday when I was working on the domain registrations in checkout and found a great written guide for passing props between children/parents/siblings, so I'm going to refactor this to give that a try. :D

Reply:
Glad to hear you found it! :)